client/ionic/src/App.tsx

```diff
@@ -32,6 +33,11 @@ const App: React.FC = () => (
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
         <Route path="/menu" component={Menu} exact={true} />
+        <Route
+          path="/protect_yourself"
```

Comment:
recognizing that this is totally a personal preference thing, but I prefer `kebab-case` to `snake_case` in URL paths. open to discussion though

Reply:
i'd say it matters.  WHO uses kebab-case on their website.  I don't have a preference, just python baked into my brain

Reply:
these paths can't be seen by the user (at least if we're not a true webapp) so meh. but  i like kebabs too (and kebab case ;)

Reply:
intention is to get scaffolds in for all pages.  closing this pr, 

reopened as https://github.com/WorldHealthOrganization/app/pull/141

for branch name, also, switching to dynamic content.

Reply:
will also update link in slack